insights_dart/test/insights_test.dart

```diff
@@ -35,11 +35,11 @@ void main() {
         userTokenStorage,
       )
         ..userTokenLeaseTime = 0
-        ..trackClick(
+        ..clickedFilters(
```

Comment:
we should also test the other methods in this file (or else one method per file)

Reply:
Done